datadog_checks_dev/README.md

```diff
@@ -0,0 +1,555 @@
+# Datadog Checks Dev
+
+[![Latest PyPI version](https://img.shields.io/pypi/v/datadog-checks-dev.svg)](https://pypi.org/project/datadog-checks-dev)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/datadog-checks-dev.svg)](https://pypi.org/project/datadog-checks-dev)
+[![License](https://img.shields.io/pypi/l/datadog-checks-dev.svg)](https://choosealicense.com/licenses)
+
+-----
+
+This is the developer toolkit designed for use by any [Agent-based][1] check or
+integration repository. There are 2 layers: one purely for development, and the
+other for high level tasks such as releasing, dependency management, etc.
+
+`datadog-checks-dev` is distributed on [PyPI](https://pypi.org) as a universal wheel
+and is available on Linux, macOS, and Windows, and supports Python 2.7/3.5+ and PyPy.
+
+**Table of Contents**
+
+- [Development](#development)
+  * [Installation](#installation)
+  * [Usage](#usage)
+- [Management](#management)
+  * [Installation](#installation-1)
+  * [Usage](#usage-1)
+    + [Clean](#clean)
+    + [Config](#config)
+      - [Find](#find)
+      - [Restore](#restore)
+      - [Set](#set)
+      - [Show](#show)
+      - [Update](#update)
+    + [Dep](#dep)
+      - [Freeze](#freeze)
+      - [Pin](#pin)
+      - [Resolve](#resolve)
+      - [Verify](#verify)
+    + [Manifest](#manifest)
+      - [Set](#set-1)
+      - [Verify](#verify-1)
+    + [Release](#release)
+      - [Changelog](#changelog)
+      - [Freeze](#freeze-1)
+      - [Make](#make)
+      - [Show](#show-1)
+        * [Changes](#changes)
+        * [Ready](#ready)
+      - [Tag](#tag)
+      - [Upload](#upload)
+    + [Test](#test)
+
+## Development
+
+This is the layer intended to be used directly for testing.
+
+### Installation
+
+```console
+$ pip install datadog-checks-dev
+```
+
+This comes with `pytest`, `mock`, and other core requirements for testing.
+Most likely you will not install this manually, but rather list it as the
+only dependency in a check's `requirements-dev.txt`.
+
+### Usage
+
+Aside from being a source for test dependencies, this also provides many useful
+utilities and fixtures (soon!) to avoid re-inventing the wheel. They can all be
+found [here][2].
+
+Some examples:
```

Comment:
Examples are not very clear here, there's no introduction about what the snippets are supposed to do, and all the code flows in the same snippet. I'd consider removing these altogether for now and make a plan about docs.